tests/e2e/core-tests/specs/shopper/front-end-checkout-create-account.test.js

```diff
@@ -45,7 +46,7 @@ const runCheckoutCreateAccountTest = () => {
 
 			// Add simple product to cart and proceed to checkout
 			await shopper.goToShop();
-			await shopper.addToCartFromShopPage(simpleProductName);
+			await shopper.addToCartFromShopPage( productId );
```

Comment:
We can drop the `simpleProductName` variable declaration on line 25 now as it's unused in this file.

Reply:
Removed in https://github.com/woocommerce/woocommerce/pull/30822/commits/ef778b898599d50e3775bbdc5909146cb89e4bfa